### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by SaeqaSultani**
**Create models**
- Create model classes for all entities as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
- Set up associations between models.
         Remember that author_id column in posts table should be the foreign_key for the users table.
**Use models to insert data**
- Familiarize yourself with ['rails c' command](https://guides.rubyonrails.org/command_line.html#bin-rails-console).
- Open rails console for your project.
- Create at least one user by running the following code:
   first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
   second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
- Create at least 4 posts written by one of the users you created by running the following code:
   first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')
- Create at least 6 posts comments for one of the posts you created by running the following code:
   Comment.create(post: first_post, author: second_user, text: 'Hi Tom!' )
- Use other [CRUD methods](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/active_record_crud.md) to find, update and delete entities.

**Create custom methods**
Your models should include some additional methods.

- Users
     A method that returns the 3 most recent posts for a given user.
- Posts
     A method that updates the posts counter for a user.
    A method which returns the 5 most recent comments for a given post.
- Comments
    A method that updates the comments counter for a post.
- Likes
    A method that updates the likes counter for a post.

### CODE REVIEW

**Review by BenMukebo — COMMENTED** [review]
## STATUS: INVALID ❌ 

Hi  @SaeqaSultani,

I can't miss felicitating your efforts and energy, you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 


- [ ] You have submitted a project, that does not have a professional Readme file make sure no text description from the template is present, all section are required except the FAQ section kindly follow this [microverse template](https://github.com/microverseinc/readme-template) and work on your README file the request another review. Make sure also you have a descriptive  PR decription and title


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@BenMukebo) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,224 @@
...
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rotimiazeez) [inline README.md]: Please ensure the provided MIT license link is valid, as the current one is broken. 👍🏽
Screenshot from 2023-04-01 14-51-28

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @SaeqaSultani 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**